agent/php_nrini.c

```diff
@@ -222,6 +223,53 @@
 
 ZEND_DECLARE_MODULE_GLOBALS(newrelic)
 
+#define NR_INI_PREFIX_LEN 9
+
+char* nr_ini_to_env(const char* ini_name) {
+  char* env_name = NULL;
+  char* buf = NULL;
+  char* ini_upper = NULL;
+  const char* NR_PREFIX_STR = "NEW_RELIC";
+  int ini_len = nr_strlen(ini_name);
+
+  // 'newrelic.' is 9 characters - anything less should be rejected.
+  if (NULL == ini_name || nr_strlen(ini_name) <= NR_INI_PREFIX_LEN) {
+    return NULL;
+  }
+
+  // ini value should start with the 'newrelic.' prefix
+  if (0 != nr_stridx(ini_name, "newrelic.")) {
```

Comment:
A more direct test is something like used in `php_environment.c::nr_php_process_environment_variable_to_nrobj()`:

```
  if (0 == nr_strncmp(ini_name, NR_PREFIX_STR, NR_INI_PREFIX_LEN)) {
```

Reply:
This will not quite work because the `ini_name` in this case should be something with the format `newrelic.<field>.<etc>`, whereas `NR_PREFIX_STR` is the "env" version of the ini, and is set to `NEW_RELIC`.

Reply:
Ok I see I have confused the use of prefix for `newrelic.` and `NEW_RELIC`.  The current form is OK.

```